client/clientimpl_test.go

```diff
@@ -93,7 +93,7 @@ func eventually(t *testing.T, f func() bool) {
 	assert.Eventually(t, f, 5*time.Second, 10*time.Millisecond)
 }
 
-func newInstanceUid(t *testing.T) types.InstanceUid {
+func genNewInstanceUid(t *testing.T) types.InstanceUid {
```

Comment:
This change is related to making `TestAgentIdentification` more realistic; the test has a `newInstanceUid := newInstanceUid(t)` statement in which the function shadows the variable of the same name, rendering the test harder to follow, needing more variables etc.

I'll see if there's an easy way to fix it, otherwise I can revert changes and leave that test as-is.

Reply:
That's fine, keep the change.